test/cpu_fusion.cpp

```diff
@@ -1003,46 +1003,90 @@ TEST(cpu_fusion, fuse_conv_relu)
+    auto external = manager->compile(f);
     auto backend = manager->allocate_backend();
+    auto cf = backend->make_call_frame(external);
 
-    auto _a = backend->make_primary_tensor_view(element::f32, shape_a);
-    vector<float> va{
-        1.25f,  2.25f, 5.25f,  6.25f, -1.25f, -1.25f, 3.25f,  -4.25f, 7.25f,  8.25f, -1.25f, -1.25f,
-        1.25f,  2.25f, -3.25f, 2.25f, 4.25f,  4.25f,  1.25f,  2.25f,  -4.25f, 2.25f, 4.25f,  4.25f,
-        0.f,    0.f,   -1.f,   0.f,   2.f,    2.f,    0.f,    0.f,    0.f,    0.f,   2.f,    2.f,
-        1.25f,  2.25f, 5.25f,  6.25f, 1.25f,  1.25f,  3.25f,  4.25f,  -7.25f, 8.25f, 1.25f,  -1.25f,
-        -1.25f, 2.25f, 3.25f,  2.25f, -4.25f, -4.25f, -1.25f, -2.25f, 4.25f,  2.25f, 4.25f,  4.25f,
-        0.f,    0.f,   1.f,    0.f,   -2.f,   2.f,    0.f,    0.f,    0.f,    0.f,   -2.f,   -2.f};
+    auto parms = f->get_parameters();
 
-    copy_data(_a, va);
+    if (parms.size() != args.size())
+    {
+        throw ngraph_error("number of parameters and arguments don't match");
+    }
 
-    auto _weights = backend->make_primary_tensor_view(element::f32, shape_weights);
-    copy_data(_weights, vector<float>{2., 2., 2., 2.});
+    std::vector<std::shared_ptr<ngraph::runtime::TensorView>> arg_tensors(args.size());
+    for (size_t i = 0; i < args.size(); i++)
+    {
+        auto t = backend->make_primary_tensor_view(parms.at(i)->get_element_type(),
+                                                   parms.at(i)->get_shape());
+        copy_data(t, args.at(i));
+        arg_tensors.at(i) = t;
+    }
 
-    auto f = make_shared<Function>(NodeVector{conv_relu, relu}, op::ParameterVector{A, weights});
+    auto results = f->get_results();
+    std::vector<std::shared_ptr<ngraph::runtime::TensorView>> result_tensors(args.size());
 
-    auto external = manager->compile(f);
-    auto cf = backend->make_call_frame(external);
+    for (size_t i = 0; i < results.size(); i++)
+    {
+        result_tensors.at(i) = backend->make_primary_tensor_view(results.at(i)->get_element_type(),
+                                                                 results.at(i)->get_shape());
+    }
 
-    shared_ptr<runtime::TensorView> _conv_relu =
-        backend->make_primary_tensor_view(element::f32, conv_relu->get_shape());
+    cf->call(result_tensors, arg_tensors);
 
-    shared_ptr<runtime::TensorView> _relu =
-        backend->make_primary_tensor_view(element::f32, relu->get_shape());
+    std::vector<std::vector<T>> result_vectors;
+    for (auto rt : result_tensors)
+    {
+        result_vectors.push_back(read_vector<T>(rt));
+    }
+    return result_vectors;
+}
+
+TEST(cpu_fusion, conv_relu_n2c1h2w2_2)
+{
+    Shape shape_a{2, 1, 6, 6};
+    Shape shape_weights{1, 1, 2, 2};
 
-    cf->call({_conv_relu, _relu}, {_a, _weights});
-    EXPECT_TRUE(test::all_close(read_vector<float>(_conv_relu), read_vector<float>(_relu)));
+    auto make_int_function = [shape_a, shape_weights]() {
```

Comment:
may be better to use full name here, make_interpreter_function. ? :)

Reply:
@pruthviIntel , actually, it looks so aligned everywhere else 😄 
```cpp
    auto int_results = execute(int_f, args, "INTERPRETER");
    auto cpu_results = execute(cpu_f, args, "CPU");
```

how strongly do you feel about this?

Reply:
hmm, true!. its okay, then.